tethys/Utils/Math.py

```diff
@@ -35,7 +37,12 @@ def SizeC(l):
 def sub2ind(arraySize,rowSub,colSub):
     linearInd  = []
     if len(rowSub) != len(colSub):
-        print 'def sub2ind at Rearranging: length of rowSub is not equal to length of colSub!'
+        ## We are logging this as a warning and ofrging ahead, but this is arguably an

```

Comment:
I agree that this should raise an error, this could cause some damage

Reply:
I'll convert that to an error.